hack/deploy-setup.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+# This script inits an elasticsearch-operator
+# to deploy an Elasticsearch cluster.  It assumes it is capable of login as a
+# user who has the cluster-admin role
+
+set -e
+
+source "$(dirname $0)/common"
+
+load_manifest() {
+  local repo=$1
+  local namespace=${2:-}
+  if [ -n "${namespace}" ] ; then
+    namespace="-n ${namespace}"
+  fi
+
+  pushd ${repo}/manifests
+    for m in $(ls); do
+      if [ "$(echo ${EXCLUSIONS[@]} | grep -o ${m} | wc -w)" == "0" ] ; then
+        oc create -f ${m} ${namespace:-} ||:
+      fi
+    done
+  popd
+}
+
+oc create namespace ${NAMESPACE} ||:
+load_manifest ${repo_dir} ${NAMESPACE}
+
+#hack openshift-monitoring
+pushd vendor/github.com/coreos/prometheus-operator/example/prometheus-operator-crd
+  for file in prometheusrule.crd.yaml servicemonitor.crd.yaml; do 
+    oc create -n ${NAMESPACE} -f ${file} ||:
+  done
+popd
+
+oc create -f hack/prometheus-operator-crd-cluster-roles.yaml ||:
+
+oc create clusterrolebinding elasticsearch-operator-prometheus-rolebinding \
+    --serviceaccount=${NAMESPACE}:elasticsearch-operator \
+    --clusterrole=prometheus-crd-edit ||:
+
+# This is necessary for running the operator with go run
```

Comment:
This should only be necessary for `Cluster Logging Operator`. Elasticsearch Operator doesn't leverage a working dir does it?